README.md

```diff
@@ -18,6 +21,9 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+Our mission is to create an object that saves us time by removing a normal task. More specifically, we want to create a washing machine that automatically orders new detergent when it is running low.
```

Comment:
Great, this is something we don't have a problem doing ourselves, but why not have the machine do it for us?